docs/UserGuide.md

```diff
@@ -281,14 +284,44 @@ If your changes to the data file make its format invalid, FoodWhere will discard
 | **slist** | `slist`                                                                                                      |
-| **sfind** | `sfind n/NAME_KEYWORD [MORE_KEYWORDS]… t/TAG_KEYWORD [MORE_KEYWORDS]…` <br> e.g., `rfind n/eatery`           |
+| **sfind** | `sfind n/KEYWORD [KEYWORD]… t/KEYWORD [KEYWORD]…` <br> e.g., `rfind n/eatery`                                |
 | **ssort** | `ssort CRITERION` <br> e.g., `ssort reversedname`                                                            |
+
+## Accepted command arguments
+
+| Argument       | Format                                                                                                      |
+|----------------|-------------------------------------------------------------------------------------------------------------|
+| `ADDRESS`      | Any ASCII text, case insensitive                                                                            |
+| `CONTENT`      | Any ASCII text                                                                                              |
+| `CRITERION`    | Valid criteria are stated at the [sfind](#finding-a-stall-sfind), [rfind](#finding-a-review-rfind) commands |
+| `DATE`         | A date in the format DD/MM/YYYY, D/MM/YYYY, DD/M/YYYY, D/M/YYYY, or with dashes instead of slashes          |
+| `EDIT_TAG`     | A possibly empty alphanumeric token without spaces, case insensitive                                        |
+| `KEYWORD`      | An alphanumeric token without spaces, case insensitive                                                      |
+| `NAME`         | A nonempty alphanumeric string with spaces, case insensitive                                                |
+| `RATING`       | An integer from 0 to 5                                                                                      |
+| `REVIEW_INDEX` | A positive integer from 1 and the number of reviews in the review list, inclusive                           |
+| `STALL_INDEX`  | A positive integer from 1 and the number of stalls in the stall list, inclusive                             |
+| `TAGS`         | A nonempty alphanumeric token without spaces, case insensitive                                              |
+
+## Future features (unimplemented)
+
+Attempt to use the following features could result in user confusion, an error message or accidentally typing in an unexpected valid command.
+
+* Copying stalls using commands to allow them to be pasted in an external application.
+  * This would need to be handled for each field, which would need significant additional effort.
+* Showing reviews/stalls that match more of the `sfind`/`rfind` search terms first
+* Finding reviews by rating
+* Finding reviews/stalls by address
+* A second window that gives more space for long text to be viewed completely
+* Undo previous command
+* Smarter error detection which detects the first error a user makes in a command
+* Clearing all reviews only
```

Comment:
Resolves #296. Perhaps you could also make it clearer in our UG that currently our clear we have now clear stalls which in turn clears reviews associated with it? If not there might be a further tester asking for a separate clear stalls command.

Reply:
How do you propose this be made clearer - since it's already specified under the clear command?

Reply:
Just thinking since all our command related to data (Stalls and Reviews) have two separate commands with the leading s or r keywords, users might expect a pair of clear commands, sclear and rclear. If we state under our current clear command that it clears stalls which in turns clears the associated reviews, might this prevents future testers from finding fault with it?

Reply:
Currently the [docs](https://ay2223s1-cs2103-w14-2.github.io/tp/UserGuide.html#clearing-all-entries--clear) state

```
Clearing all entries: clear
Deletes all entries from the stall list and review list.
```

Are you proposing something like
```
Clearing all entries: clear
Clears FoodWhere by removing all stalls and the associated reviews.
```

Reply:
Yup that looks good, do you think it will be clearer to the testers why we don't need a separate sclear command after we change to this new phrasing?

Reply:
As the testers are going to read the DG as well, I agree this would support their concept on how the model works or help them understand the model. I don't think this improves the understanding for users who don't read the developer guide. However, since it is a net positive, I would implement the suggestion.